Slider/Slider.jsx

```diff
@@ -0,0 +1,79 @@
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+import { MDCSlider } from "@material/slider";
+
+/**
+ * @prop disabled = false
+ */
+export default class Slider extends MaterialComponent {
+  constructor() {
+    super();
+    this.componentName = "slider";
+    this._mdcProps = ["disabled", "min", "max", "value", "step"];
```

Comment:
So the way _mdcProps works is:
if user gives the value as true/false then these will be converted to specific mdc_componentname--classname

https://github.com/prateekbh/preact-material-components/blob/master/MaterialComponent.js#L35-L37

So I dont this these are needed here